identity/types.go

```diff
@@ -4,7 +4,7 @@ type DidDoc struct {
 	Context             []string                   `json:"@context"`
 	Id                  string                     `json:"id"`
 	AlsoKnownAs         []string                   `json:"alsoKnownAs"`
-	VerificationMethods []DidDocVerificationMethod `json:"verificationMethods"`
+	VerificationMethods []DidDocVerificationMethod `json:"verificationMethod"`
```

Comment:
This appears to be a copy paste error, it's singular in the did document and plural in the did data